### PR TITLE
Expose OOM crash in UI

### PR DESCRIPTION
**Pull request by Jamminroot**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [X] Are tests passing locally?
* [X] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [X] Did you check UI tests on the sample app? They are not executed on CI.
* [X] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description
[AB#65508](https://dev.azure.com/msmobilecenter/454a20a9-dfe6-4c1f-8ae8-417f76adb472/_workitems/edit/65508)

### CODE REVIEW

File: CrashLib/CrashLib/MSCrashOOM.mm [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  return @""
+          "Execute an infinite loop with excessive memory allocation, which "
+          "causes a OS to terminate app.";
+}
+
+- (void)crash {
+  int arrSize = 128 * 1024 * 1024;
+  while (true) {
+    volatile auto newArr = new char[arrSize];
+    std::fill(newArr, newArr + arrSize, 42);
```
Comment (MatkovIvan) [inline CrashLib/CrashLib/MSCrashOOM.mm]: С++ isn't required here

File: CrashLib/CrashLib/MSCrashOutOfMemory.mm [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  return @"Memory";
+}
+
+- (NSString *)title {
+  return @"Produce memory shortage (OOM)";
+}
+
+- (NSString *)desc {
+  return @""
+          "Execute an infinite loop with excessive memory allocation, which "
```
Comment (jaelim-ms) [inline CrashLib/CrashLib/MSCrashOutOfMemory.mm]:
```suggestion
          "Execute an infinite loop with excessive memory allocation which "
```

File: CrashLib/CrashLib/MSCrashOutOfMemory.mm [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+          "causes a OS to terminate app.";
```
Comment (jaelim-ms) [inline CrashLib/CrashLib/MSCrashOutOfMemory.mm]:
```suggestion
          "causes an OS to terminate app.";
```

File: Sasquatch/Sasquatch/ViewControllers/MSCrashesViewController.swift [hunk 4]
```diff
@@ -178,7 +181,7 @@ class MSCrashesViewController: UITableViewController, UIImagePickerControllerDel
         alertController.addAction(cancelAction)
         alertController.addAction(clearAction)
         self.present(alertController, animated: true, completion: nil)
-      }
+      }
```
Comment (russelarms) [inline Sasquatch/Sasquatch/ViewControllers/MSCrashesViewController.swift]:
```suggestion
      }
```

**Pull request merged**